website/.spelling

```diff
@@ -1210,18 +1210,17 @@ taskId
 taskid
 un
  - ../docs/operations/basic-cluster-tuning.md
-100MB
-128MB
+100MiB
```

Comment:
too bad spelling system doesn't have a better way of handling all these like `KiB, GiB` :/

Reply:
Yes, spelling check is not so smart, it treats `100MiB` as a whole word. Suppression of `MiB` won't apply to it.